streams.json

```diff
@@ -6758,141 +8900,189 @@
               "pskopek@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/jbossas/redhat-picketbox",
+          "repository_url": "https://github.com/picketbox/picketbox.git",
           "codebase": "master",
-          "tag": "5.0.2.Final",
+          "tag": "5.0.2.Final-redhat-1",
+          "group_id": "org.picketbox",
+          "artifact_id": "picketbox-aggregator-pom",
           "version": "5.0.2.Final-redhat-1",
-          "gav": "org.picketbox",
-          "comment": " There is eap-7.x(4.9.x) branch, but current 7.x is at master. No tag, last is 5.0.0A3, 7.x is at A3"
+          "gav": "picketbox-aggregator-pom:org.picketbox:5.0.2.Final-redhat-1",
+          "comment": "TAG suspicious. TAG in pom: jboss-parent-19, TAG in mapping tool: 5.0.2.Final-redhat-1.  There is eap-7.x(4.9.x) branch, but current 7.x is at master. No tag, last is 5.0.0A3, 7.x is at A3"
         },
         {
           "component_name": "picketbox-commons",
           "contacts": [
               "pskopek@redhat.com"
           ],
           "repository_type": "SVN",
-          "repository_url": "http://anonsvn.jboss.org/repos/picketbox/commons",
+          "repository_url": "http://anonsvn.jboss.org/repos/picketbox/commons/",
           "codebase": "trunk",
-          "tag": "tags/1.0.0.final",
+          "tag": "1.0.0.final",
+          "group_id": "org.picketbox",
+          "artifact_id": "picketbox-commons",
           "version": "1.0.0.final-redhat-5",
-          "gav": "org.picketbox",
-          "comment": ""
+          "gav": "picketbox-commons:org.picketbox:1.0.0.final-redhat-5",
+          "comment": "TAG suspicious. TAG in pom: jboss-parent-16, TAG in mapping tool: 1.0.0.final. "
         },
         {
           "component_name": "picketlink-bindings-25",
```

Comment:
It looks like a duplicated by jbossas-redhat-picketlink-bindings found in prod KB. I suggest leaving picketlink-bindings-25 out of the final list